full-stack-tests/rpc/src/backend/http.ts

```diff
@@ -35,6 +37,10 @@ async function init() {
 
   // eslint-disable-next-line no-console
   console.log(`Mobile backend for rpc full-stack-tests listening on port ${mobilePort}`);
+  return () => {
+    httpServer.close();
+    MobileHost.onEnterBackground.raiseEvent();
```

Comment:
@swbsi Is there a better way to shutdown the `MobileRpcServer`?  This doesn't seem right...

Reply:
Yeah, but how would I grab the `MobileRpcServer`?  Would it make sense to add something like a `MobileHost.shutdown()`?

Reply:
The event is just there to make that OS lifecycle callback available to anyone that needs to know.

We don’t currently need to have a shutdown function...the app simply exits.

For testing scenarios like this, it would make sense.

Reply:
OK, I'll leave this as-is for now and someone more familiar with the mobile setup can cleanup/refactor as-needed.

cc: @khanaffan